pep-0570.rst

```diff
@@ -19,27 +19,33 @@ Abstract
 ========
 
 This PEP proposes to introduce a new syntax, ``/``, for specifying
-positional-only parameters in Python.
+positional-only parameters in Python function signatures.
```

Comment:
@willingc in #972, I changed all references to "function signature" to "function definition" to be congruent with the text in https://docs.python.org/3/tutorial/controlflow.html#defining-functions

> The keyword `def` introduces a function definition. [...]

For consistency with the rest of the document, does it make sense to keep with "definition" over "signature"?  There are two other references to "signature" further down…

Reply:
Thanks for the thorough review @ericvw :snake: Yeah, we should be consistent throughout. I'll take a look on my flight and make some edits. FYI - keep reviewing PRs, you are doing good work.